src/eckit/maths/CMakeLists.txt

```diff
@@ -1,16 +1,24 @@
-list( APPEND eckit_maths_lib_srcs
-Eigen.h
-Lapack.h
-Lapack.cc
-Matrix.h
-MatrixEigen.h
-MatrixLapack.h
-)
+list(APPEND eckit_maths_public_libs eckit)
+list(APPEND eckit_maths_sources
+    Eigen.h
+    Lapack.h
+    Lapack.cc
+    Matrix.h
+    MatrixEigen.h
+    MatrixLapack.h)
+
+if(eckit_HAVE_CONVEX_HULL)
+    list(APPEND eckit_maths_sources ConvexHull.h ConvexHullN.h Qhull.cc Qhull.h)
+    list(APPEND eckit_maths_public_libs Qhull::Qhull)
```

Comment:
With previous comment about private includes within code, this could become private instead of public

Reply:
Yes I'm aware, will fix this